src/components/map/index.tsx

```diff
@@ -327,45 +326,38 @@ class DeckGLMap extends React.Component {
     }
   }
 
-  _onClick(x, y, object) {
+  async selectTree(treeId: string) {
     const { setViewport } = this.props;
```

Comment:
Grabbing this from the `Action(store)` initialisation didn't properly work as it seems that the `connect` fuction is doing some magic (applying the current state to the first argument of the `setViewport` function in `Actions.tsx`).
This is why I import `setViewport` from the props and `getTree` from `Actions(store)`. This should be cleaned up in the future.

Reply:
Okay.